roles/kubernetes-apps/ansible/templates/coredns-deployment.yml.j2

```diff
@@ -95,15 +100,6 @@ spec:
           timeoutSeconds: 5
           successThreshold: 1
           failureThreshold: 5
-        readinessProbe:
```

Comment:
Losing this probe seems bad. We don't want to allow clients to do dns queries against a server that is definitely not healthy. If anything, change the probe to be this:
```
          httpGet:
            path: /health
            port: 8080
            scheme: HTTP
          initialDelaySeconds: 0
          timeoutSeconds: 5
          successThreshold: 1
          failureThreshold: 15
```